sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -2978,6 +3008,29 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
      if(debugprint)print*,'sample stc = ',1,stc(isa,jsa,9)
 
       END IF
+!
+! E. James - 27 Sep 2022: this is for RRFS, adding smoke and dust
+! extinction; it needs to be after ZINT is defined.
+!
+      if (modelname == 'FV3R') then
+       do l = 1, lm
+        do j = jsta_2l, jend_2u
+         do i = ista_2l, iend_2u
+            taod5503d ( i, j, l) = extsmoke ( i, j, l ) + extdust ( i, j, l )
+            dz = ZINT( i, j, l ) - ZINT( i, j, l+1 )
+            aextc55 ( i, j, l ) = taod5503d ( i, j, l ) / dz
+         if(i==im/2.and.j==(jsta+jend)/2)print*,'sample taod5503d= ',   &
```

Comment:
@EricJames-NOAA Could you also add a threshold "debugprint" for these three print statements? 
e.g. if(debugprint.and.i==im/2.and.j==(jsta+jend)/2)print*,